.gitignore

```diff
@@ -8,3 +8,4 @@ target/
 /cli/pack/
 /.idea/
 /.idea_modules/
+bin/
```

Comment:
Oh... I see we have the idea ignores as well. Ok fair enough. However, you should probably anchor them at the top level.